cluster-api-gcp/cloud/google/serviceaccount.go

```diff
@@ -21,14 +21,15 @@ import (
 	"os/exec"
 
 	"github.com/golang/glog"
-	clusterv1 "k8s.io/kube-deploy/cluster-api/api/cluster/v1alpha1"
 	"k8s.io/kube-deploy/cluster-api-gcp/util"
+	clusterv1 "k8s.io/kube-deploy/cluster-api/api/cluster/v1alpha1"
 )
 
 const (
 	ServiceAccountPrefix    = "k8s-machine-controller-"
 	ServiceAccount          = "service-account"
 	MachineControllerSecret = "machine-controller-credential"
+	ServiceAccountNs        = "kube-system"
```

Comment:
This should be sth configurable in the future. It doesn't mean you should change this. Just FYI

Reply:
Agreed! Filed to track it: https://github.com/kubernetes/kube-deploy/issues/573